nautobot_golden_config/jobs.py

```diff
@@ -150,7 +150,15 @@ def run(self, data, commit):
 
         # Commit / Push each repo after job is completed.
         for intended_repo in intended_repos:
-            self.log_debug(f"Push new intended configs to repo {intended_repo.url}.")
+            self.logger.info(
+                f'{intended_repo["repo_obj"].nautobot_repo_obj.name}: the new Git repository hash is "{intended_repo["repo_obj"].head}"',
+                extra={
+                    "grouping": "GC Repo Commit and Push",
+                    "object": intended_repo["repo_obj"].nautobot_repo_obj.name,
+                },
+            )
+
+            self.log_debug(f"Push new intended configs to repo {intended_repo['repo_obj'].nautobot_repo_obj.name}.")
```

Comment:
I'm not seeing `repo_obj` on the `nautobot_golden_config.utilities.git.GitRepo` class and I'm not seeing `nautobot_repo_obj` anywhere in GoldenConfig or Nautobot v1.6. It looks to me like this should be `intended_repo.obj.name` and `intended_repo.obj.current_head`?